### ISSUE TITLE
Add repetitition penalty logits processor

### ISSUE THREAD

**Issue by rlouf** [outlines-dev/outlines#647]
Repetition penalty decreases the score of tokens that have already been generated. It should be simple to implement.
